joomla/joomla-cms
# [com_content] review config global default options

### Summary of Changes

This PR intends to normalize onfig global default options in com_content.

The method used was:
1. Review cs of the com_content config.xml file - also add missing default values
2. Check the current params in installation joomla.sql mysql
3. Add missing defaults to install sql

### Testing Instructions

0. Code review
1. Apply patch
2. Delete configuration.php
3. Install Joomla (no sample)
4. Create an article and check all "Use Global" have the global value.

### Documentation Changes Required

None.

### Added/changed defaults in the xml

changed:
- targeta: 0 (was Parent that didn't exist in list options)
- targetb: 0 (was Parent that didn't exist in list options)
- targetc: 0 (was Parent that didn't exist in list options)

added:
- float_intro: left
- float_fulltext: left
- date_format: *empty*
